docs/pages/kicon.vue

```diff
@@ -2,7 +2,7 @@
 
   <DocsPageTemplate apiDocs>
     <DocsPageSection title="Overview" anchor="#overview">
-      See <DocsInternalLink href="icons" text="the page on icons" /> for design guidance and a list of available icons. If an invalid icon is used, in development Vue.js validation will warn about the error. The icon will display as a broken image <KIcon icon="brokenImage" /> icon.
+      See <DocsInternalLink href="/icons" text="the page on icons" /> for design guidance and a list of available icons. If an invalid icon is used, in development Vue.js validation will warn about the error. The icon will display as a broken image <KIcon icon="brokenImage" /> icon.
```

Comment:
There are (at least) a couple other places where this issue seems to exist:

https://github.com/learningequality/kolibri-design-system/blob/37dcf4f8db24d0afc5a72586bb35771dd29c8b1d/docs/pages/ktextbox.vue#L5

https://github.com/learningequality/kolibri-design-system/blob/37dcf4f8db24d0afc5a72586bb35771dd29c8b1d/docs/pages/kmodal.vue#L5
 

Could possibly add some validation and/or fallback behavior to the `DocsInternalLink` component? Looks like most `href` attributes include a preceding `/` and this is the technically correct behavior.

Reply:
This change was lost. Added a PR here: https://github.com/learningequality/kolibri-design-system/pull/462